Feat(impl)/225 use caching for discovery finder requests (#693)
catenax-ng/tx-item-relationship-service: docs/src/docs/administration/configuration.adoc

The EDC consumer controlplane endpoint URL for data management, including the protocol.
If left empty, this defaults to the internal endpoint of the controlplane provided by the irs-edc-consumer Helm chart.

==== <discoveryFinderClient.cacheTTL>

> When IRS calls the DTR discovery finder URL for BPNL's there is a cache mechanism between them, to improve performance.

Reviewer: ```suggestion
When IRS calls the Discovery Finder URL for BPNLs, the results are cached to improve performance.
```

This parameter defines how long the cache is maintained before it is cleared.
Data is in ISO 8601.

==== <connectorEndpointService.cacheTTL>

> When IRS calls EDC Discovery Service to fetch connector endpoints for BPNL's there is a cache mechanism between them, to improve performance.

Reviewer: ```suggestion
When IRS calls EDC Discovery Service to fetch connector endpoints for BPNLs, the results are cached to improve performance.
```
